airframe-control/src/main/scala/wvlet/airframe/control/ResultClass.scala

```diff
@@ -25,7 +25,7 @@ object ResultClass {
   /**
     * A label for the successful code execution result
     */
-  case object Succeeded extends ResultClass
+  case class Succeeded(value: Any) extends ResultClass
```

Comment:
This will extend the responsibility of the result classifier to generate the default result to convert Failed(exception) into Succeeded(value), so if we add this, the class will no longer ResultClass. 

Ideally, we should minimize the responsibility of the result classifier only to _classify_ result types, and let the failback handler produce the default response for certain types of errors.

Reply:
I see. Modified errorClassifier can return only `ResultClass.Failed` so that there is no necessity to have a result value at ResultClass.